lib/index.js

```diff
@@ -45,10 +46,23 @@ function compile(
   let resName
 
   try {
-    const [moduleName, bsCode] = Retyped.compile(filename, source, debugMode)
+    const [moduleName, bsCode, diagnosticErrors] = Retyped.compile(filename, source, debugMode)
     const fmtCode = format(bsCode)
     res = fmtCode
     resName = moduleName
+    if (diagnosticErrors.length > 0) {
```

Comment:
Can this check be moved to `cli.js`? `index.js` is the library entry point and can have consumers other than CLI. If something else is using ReasonablyTyped we wouldn't want to log to the console in the case of an error 🙂

Reply:
Of course yes, that's make sense :+1: 
However, could you let me know what a `includeModule` flag of `compile` function suit for :question: 
When this flag is `false`, It seems `compile` function returns string to `cli.js` even though `cli.js` seems to expects tuple of moduleName and bsCode :thinking:

Reply:
`includeModule` returns the module name generated along with the generated code.

I'm ok with changing `index.js` to return an object like:

```js
{
  moduleName: string,
  source: string,
  errors: diagnosticError[].
}
```

Does that help?

Reply:
That sounds would be helpful, I'd say understood purpose of `includeModule`, thanks :pray: